test/evergreen.yml

```diff
@@ -134,6 +147,16 @@ functions:
         for i in $(seq ${times|1}); do
           ./t -1 -c ${config|../../../test/format/CONFIG.stress} ${extra_args|}
         done
+  "format test script":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/test/format"
+      script: |
+        set -o errexit
+        set -o verbose
+        for i in $(seq ${times|1}); do
+          ${test_env_vars|} ./format.sh ${format_test_script_args|} ${smp_command|} 2>&1
```

Comment:
`format.sh` has `[format-configuration]` as it's last command argument, which is used to pass format program arguments e.g. `timer=5 run=4`. Is `format_test_script_args` going to cover `[format-configuration] argument`? If yes the `${smp_command|}` part needs to be moved forward.

Reply:
Good catch. Will move the `${smp_command|}` option